src/task/task.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package taskapi;
+option go_package = "github.com/pachyderm/pachyderm/v2/src/task";
+
+import "gogoproto/gogo.proto";
+
+enum State {
+  UNKNOWN = 0;
+  RUNNING = 1;
+  SUCCESS = 2;
+  FAILURE = 3;
+  CLAIMED = 4; // not a real state used by task logic
+}
+
+message TaskInfo {
```

Comment:
A couple of nitpicky things:
- Be sure not to skip any field numbers (i.e. 2). Any gaps should be set to reserved, but that should only occur when a field is being removed while maintaining compatibility.
- Collocate the input fields (type, data) in the message. So the last two could be for the input fields.

Reply:
ah, must have missed when I was switching things around